module/commands/help.py

```diff
@@ -5,13 +5,14 @@
 from module.shared import AULARIO, CLOUD, CUSicon, check_log
 
 
-def help_cmd(update: Update, context: CallbackContext):
+def help_cmd(update: Update, context: CallbackContext, **kwargs: int):
     """Called by the /help command.
```

Comment:
It works, but why use **kwargs?
I would either do something like
```python
def help_cmd(update: Update, context: CallbackContext, chat_id: int = None, message_id: int = None):
...
if(chat_id is not None and message_id is not None):
        context.bot.editMessageText(text=message_text, chat_id=kwargs['chat_id'], message_id=kwargs['message_id'], 
```
or, since we are just editing the message with a callback from the same message
```python
def help_cmd(update: Update, context: CallbackContext, edit: bool = False):
...
if (edit):
        context.bot.editMessageText(text=message_text, chat_id=update.message.chat_id, message_id=update.message.message_id)
```

Reply:
I am not very aware of the telegram data structure, I used kwargs because it's implementation doesn't change the method signatures, so all the calls of that method are still valid.

Reply:
Of course, your suggestions works too thanks to the default value for the added argument(s). They can do all the same thing, which on you want to implement depends on your code style preferences

Reply:
In the case of most handlers, like this one, the Callback signature is:
```python
def callback(update: Update, context: CallbackContext)
```
So, provided the two required parameter are present, you can add any optional parameter you want.
This said, it is just a style preference.